montepy/data_inputs/isotope.py

```diff
@@ -180,6 +180,9 @@ def mcnp_str(self):
         """
         return f"{self.ZAID}.{self.library}"
 
+    def nuclide_str(self):
+        return f"{self.element.symbol}-{self.A}.{self._library}"
+
```

Comment:
I like this. Except library can be None. or Empty. I'd add an else clause that drops the `.`.

Reply:
Good point. Just accounted for that.